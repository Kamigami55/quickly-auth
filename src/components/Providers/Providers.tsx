import { Provider as JotaiProvider } from 'jotai';

export function Providers({ children: children }: React.PropsWithChildren) {
  return <JotaiProvider>{children}</JotaiProvider>;
}
