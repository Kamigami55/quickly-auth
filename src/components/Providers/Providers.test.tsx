import '@testing-library/jest-dom';

import { render } from '@testing-library/react';

import { Providers } from './Providers';

describe('Providers', () => {
  it('renders providers unchanged', () => {
    const { container } = render(<Providers>test</Providers>);
    expect(container).toMatchSnapshot();
  });
});
