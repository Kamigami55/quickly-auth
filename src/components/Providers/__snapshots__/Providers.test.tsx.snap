// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Providers renders providers unchanged 1`] = `
<div>
  test
</div>
`;
